src/git.sh

```
@@ -1,42 +1,329 @@
-  if [ "$1" == "--store" ]; then
+  if [ "$store" = "true" ]; then
     echo "last_tag=${last_tag}"
     echo "last_tag=${last_tag}" >> $GITHUB_OUTPUT
+  else
+    echo "${last_tag}"
   fi
+}
+
+git_build_changelog() {
+  # Parse arguments using the new parse_arguments function
+  local args_json=$(parse_arguments "$@")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)